### ISSUE TITLE
Upsert doesn't work for bulkCreate

### ISSUE THREAD

**Issue by alyrik** [feathersjs-ecosystem/feathers-elasticsearch#75]
### Steps to reproduce

Pass array to `create` method with `{ upsert: true }` params.

### Expected behavior
It should create new items and update existing ones.

### Actual behavior
Existing items are not updated.

### System configuration

@feathersjs/feathers@3.3.1
feathers-elasticsearch@1.4.0
elasticsearch@15.4.1

**NodeJS version**:
v10.15.0

**Operating System**:
Debian

**Comment by othersideofphase** [comment]
I came across this issue today, setting `upsert: true` while making a bulk create request tries to call `create` on a potentially already existing record giving:
```{
  "_index": "index_name",
  "_type": "type_name",
  "_id": "2276654",
  "status": 409,
  "error": {
    "type": "version_conflict_engine_exception",
    "reason": "[type_name][2276654]: version conflict, document already exists (current version [4])",
    "index_uuid": "149AMdPPTHSo4LvC_bBrkw",
    "shard": "4",
    "index": "index_name"
  }
}```

I created a small PR for this - can anyone help test

**Comment by daffl** [comment]
Closed via #78
